FrontEnd/src/components/ProfilePage/FormComponents/AdditionalInfo.js

```diff
@@ -0,0 +1,114 @@
+import css from './FormComponents.module.css';
+import { useState, useEffect } from 'react';
+
+import HalfFormField from './FormFields/HalfFormField';
+import TextField from './FormFields/TextField';
+
+const LABELS = {
+    'foundationYear': 'Рік заснування',
+    'companySize': 'Розмір компанії',
+    'topClients': 'Топ клієнти',
+    'passedAudit': 'Пройдений аудит',
+};
+
+const AdditionalInfo = (props) => {
+    const [user, setUser] = useState(props.user);
+    const [foundationYearError, setFoundationYearError] = useState(null);
+    const maxLength = 1000;
```

Comment:
It is better to move it from the component to not create it on every render. Constants should be written in upper-case. Consider using more meaningful name like `TEXT_AREA_MAX_LENGTH`

Reply:
thanks for your explanation and example, I understood and fixed it in the new commit